torchbenchmark/models/drq/origin

```diff
@@ -0,0 +1 @@
+https://github.com/denisyarats/drq/tree/365e5969f9a748ad63d854ce8c8cc8f90c1de4e0
```

Comment:
Is this a submodule? Do we actually want this committed?

Reply:
It is not a submodule, in [ADD_MODEL]`https://github.com/pytorch/benchmark/blob/master/torchbenchmark/models/ADDING_MODELS.md` instructions we would like to add this file so that we can track where the model code is originally from.